user-guide/scheduler.rst

```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
 Resource Limits
 ---------------
 
-There are different resource limits on ARCHER2 for different purposes.
+The ARCHER2 resource limits for any given job are covered by three separate attributes.
+
+* The amount of *primary resource* you require (i.e., CPU cores).
```

Comment:
On ARCHER2, the primary resource is "nodes", not "CPU cores"

Reply:
OK